RecoLocalTracker/SiStripRecHitConverter/src/StripCPEfromTemplate.cc

```diff
@@ -126,7 +126,7 @@ StripCPEfromTemplate::localParameters( const SiStripCluster& cluster,
       //else 
       //cout << "Do not use templates for strip modules other than IB1, IB2, OB1 and OB2" << endl;
       
-      StripGeomDetUnit* stripdet = (StripGeomDetUnit*)(&det);
+      const StripGeomDetUnit* stripdet = (const StripGeomDetUnit*)(&det);
```

Comment:
In the future, I'd suggest using either ```std::static_cast<>``` or ```std::dynamic_cast<>``` since both avoid the problems of accidently casting away ```const```.